### PR TITLE
[zh-cn] sync concepts/workloads/pods/*

### PR DESCRIPTION
**Pull request by qlijin**
sync files:

      content/zh-cn/docs/concepts/workloads/pods/init-containers.md
      content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md
      content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -101,7 +101,8 @@ the application containers for the Pod and runs them as usual.
 
 Init 容器支持应用容器的全部字段和特性，包括资源限制、
 [数据卷](/zh-cn/docs/concepts/storage/volumes/)和安全设置。
-然而，Init 容器对资源请求和限制的处理稍有不同，在下面[资源](#resources)节有说明。
+然而，Init 容器对资源请求和限制的处理稍有不同，在下面
+[容器内的资源共享](#resource-sharing-within-containers)节有说明。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/init-containers.md:105]:
```suggestion
然而，Init 容器对资源请求和限制的处理稍有不同，
在下面[容器内的资源共享](#resource-sharing-within-containers)节有说明。
```
中文之间不要折行，通常在标点、数字、中英之间换行

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -462,29 +462,36 @@ When a Pod's containers are Ready but at least one custom condition is missing o
...
 <!--
-The `PodHasNetwork` condition is set to `False` by the Kubelet when it detects a
+The `PodReadyToStartContainers` condition is set to `False` by the Kubelet when it detects a
 Pod does not have a runtime sandbox with networking configured. This occurs in
 the following scenarios:
 -->
-当 kubelet 检测到 Pod 不具备配置了网络的运行时沙箱时，`PodHasNetwork` 状况将被设置为 `False`。
-以下场景中将会发生这种状况：
+当 kubelet 检测到 Pod 不具备配置了网络的运行时沙箱时，`PodReadyToStartContainers`
+状况将被设置为`False`。以下场景中将会发生这种状况：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:494]:
```suggestion
状况将被设置为 `False`。以下场景中将会发生这种状况：
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -831,16 +838,17 @@ shutdown.
...
+Once the grace period has expired, the KILL signal is sent to any remaining
+processes, and the Pod is then deleted from the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}. If the kubelet or the
+container runtime's management service is restarted while waiting for processes to terminate, the
+cluster retries from the start including the full original grace period.
 -->
-通常情况下，容器运行时会发送一个 TERM 信号到每个容器中的主进程。
-很多容器运行时都能够注意到容器镜像中 `STOPSIGNAL` 的值，并发送该信号而不是 TERM。
+通常情况下，通过 pod 的体面终止，kubelet 向容器运行时发送请求，尝试通过首先向每个容器中的主进程发送
+带有体面超时限期的 TERM（又名 SIGTERM）信号来停止 pod 中的容器。停止容器的请求由容器运行时异步处理。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:849]:
```suggestion
通常 Pod 体面终止的过程为：kubelet 先发送一个带有体面超时限期的 TERM（又名 SIGTERM）
信号到每个容器中的主进程，将请求发送到容器运行时来尝试停止 Pod 中的容器。
停止容器的这些请求由容器运行时以异步方式处理。
```

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md
```

/hold

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**